### PR TITLE
[bitnami/node] Adds service account

### LINKED ISSUE #8213: bitnami/node: Allow a ServiceAccount to be specified
**Which chart**:
bitnami/node

**Is your feature request related to a problem? Please describe.**
The chart doesn't allow a ServiceAccount to be associated with the Pods. 

**Describe the solution you'd like**
Add a `serviceAccountName` optional property to the Deployment spec

**Additional context**
I'm deploying a Node.js app to Amazon EKS. In order for this service to call AWS APIs, it must be given certain permissions. The mechanism prescribed by Amazon for granting permissions to workloads is by associating an IAM role having those permissions to a ServiceAccount and associating that ServiceAccount with the Pods. The current deployment template (in deployment.yaml) does not include  an optional `serviceAccountName` property to associate the ServiceAccount with the Pods.

### PR DESCRIPTION
**Pull request by rafariossaa**
**Description of the change**

Adds service account to the deployment

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #8213

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/node/values.yaml [hunk 1]
```diff
@@ -272,6 +272,23 @@ extraVolumes: []
...
+  ## @param serviceAccount.name The name of the ServiceAccount to use.
+  ## If not set and create is true, a name is generated using the `common.names.fullname` template
+  name: ""
+  ## @param serviceAccount.annotations Annotations for service account. Evaluated as a template.
+  ## Only used if `create` is `true`.
+  ##
+  annotations: {}
+  ## @param serviceAccount.autoMount Auto-mount the service account token in the pod
+  ##
+  autoMount: false
```
Comment (migruiz4) [inline bitnami/node/values.yaml:291]: Please use `automountServiceAccountToken` not `autoMount` as most of our charts do.

File: bitnami/node/templates/deployment.yaml [hunk 2]
```diff
@@ -23,6 +23,8 @@ spec:
         {{- include "common.tplvalues.render" (dict "value" .Values.podLabels "context" $) | nindent 8 }}
         {{- end }}
     spec:
+      automountServiceAccountToken: {{ .Values.serviceAccount.autoMount }}
```
Comment (migruiz4) [inline bitnami/node/templates/deployment.yaml:26]: Please add `automountServiceAccountToken` in the serviceAccount.yaml, not at pod level.

Adding granular `automountServiceAccountToken` at pod-level would be a different feature we do not support yet.

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Please check my suggestions.

**Review by juan131 — APPROVED** [review]
LGTM

**Review by migruiz4 — APPROVED** [review]
Thank you for applying the suggested changes!

**Pull request merged**